src/components/Navbar.js

```diff
@@ -392,6 +411,16 @@ class Navbar extends React.Component {
 						}
 					</Grid>
 					</Grid>
+					<IconButton
```

Comment:
Can you add a title to this. Something like: `title: "Report a Bug"` to Icon button. This basically shows a description when you hover over the button so users can know what its for.